### ISSUE TITLE
Update TCGA `broad_histology` and `cancer_group` in histologies.tsv 

### ISSUE THREAD

**Issue by jharenza** [PediatricOpenTargets/ticket-tracker#30]
#### What data file(s) does this issue pertain to?
histologies.tsv`

#### What release are you using?

v4

#### Put your question or report your issue here.
Update `broad_histology` and `cancer_group` in TCGA cohort to use the `short_histology` names.

**Comment by kgaonkar6** [comment]
```
v5_histology %>% filter(cohort=="TCGA") %>% pull(cancer_group) %>% unique()
 [1] "GBM"  "OV"   "LUAD" NA     "LUSC" "PRAD" "UCEC" "BLCA" "TGCT" "ESCA" "PAAD" "KIRP" "LIHC" "CESC" "SARC" "BRCA" "THYM" "MESO" "COAD" "STAD"
[21] "SKCM" "CHOL" "KIRC" "THCA" "HNSC" "LAML" "READ" "LGG"  "DLBC" "KICH" "HNS"  "UCS"  "ACC"  "PCPG" "UVM" 
```

broad_histology is long forms of short_histology

**Comment by jharenza** [comment]
closed with #5
